docs/docs/rest-api/mesosphere/marathon/api/v2/GroupsResource_show.md

```diff
@@ -1,7 +1,26 @@
-## GET `/v2/groups/{groupId}`
+## GET `/v2/groups/{groupId}?embed=...`
 
 List the application groups with id `groupId`.
 
+Embeds nested resources that match the supplied path.
+You can specify this parameter multiple times with different values.
+Unknown embed parameters are ignored.
+If you omit this parameter, it defaults to `group.groups`, `group.apps`
+
+- `group.groups` embed all child groups of each group<br/>
+
+- `group.apps` embed all apps of each group<br/>
+
+- `group.apps.tasks` embed all tasks of each application<br/>
+
+- `group.apps.counts` embed all task counts (tasksStaged, tasksRunning, tasksHealthy, tasksUnhealthy) <br/>
```

Comment:
OK – I wrote that comment because the lines below don't have the `br`s so I thought it was strange.